scripts/sanitizer-env.sh

```diff
@@ -25,5 +25,14 @@ if test -n "$OUR_PRELOAD"; then
     export LD_PRELOAD
     echo "LD_PRELOAD has been set to \"$LD_PRELOAD\"."
     echo "The sanitizer might report issues with ANY process you execute."
+else
+    echo
+    echo "WARNING: compiler returned non-existing library name $OUR_PRELOAD."
```

Comment:
it looks like this belongs to stderr?

if we use heredoc, we can simply redirect a single command to >&2

Reply:
Added extra commits to update the extra script and convert the Unknown compiler case to warn/stderr.